content/more-on-functions/reading/functions-as-values/_index.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Functions as Values"
+date: 2023-05-25T12:55:09-05:00
+draft: false
+weight: 1
+originalAuthor: John Woolbright # to be set by page creator
+originalAuthorGitHub: jwoolbright23 # to be set by page creator
+reviewer: Sally Steuterman # to be set by the page reviewer
+reviewerGitHub: gildedgardenia # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+Functions are powerful tools in any programming language, and JavaScript uses these tools in some flexible and creative ways. This chapter introduces a bit more of the power of functions.
+
+## Functions Are Data
+
+We :ref:`defined a value <def-value>` as "a specific piece of data." Some examples are the number `42`, the string `"LC101"`, and the array `["MO", "FL", "DC"]`. *Functions are also values*, and while they appear to be very different from other values we have worked with, they share many core characteristics.
```

Comment:
I'm not sure what the intent here was/is are we using coding syntax for normal words? should this be bold or italicized, underlined? 

1. Should this read values (Strings, Arrays, Boolean, null values etc.) are pieces of data that can be assigned to variables. 
2. Also in JavaScript, Functions are Objects so the headers and text Functions as Values should be updated to Functions as Objects or Classes "Functions are also values"

Reply:
Updated the textbook to reflect the above changes. Also fixed the missing relref.